backend/compat.py

```diff
@@ -1 +1,13 @@
+# Werk van Sil
 
+# for-loop
+# [] 
+
+
+def validate_appointment(data: dict):
+    KeyList = ['title', 'description', ...]
+    for key in KeyList:
+        if key not in dict:
+            return False
+        else:
+            continue
```

Comment:
Als een `for`-loop zijn 'body' (ingesprongen deel) heeft uitgevoerd gaat hij terug naar het begin van de loop zolang er nog items zijn om overt te itereren. Wat betekent dit voor deze `continue`?